### PR TITLE
E/mobile/ip

### PR DESCRIPTION
**Pull request by chaitanya-battula**
please review my programs

### CODE REVIEW

**Comment by chaitanya-battula** [comment]
PR1 code review

File: Chaitanya_1/adb_trandfer_file.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#copy a file device to computer by using adb commands
+import os
+import subprocess
+def copy_a_file_device_to_computer(cmd_list):
+      try:
+            proc = subprocess.check_output(cmd).decode('utf-8')
+      except Exception as e:
+            print("Error :{}".format(str(e)))
+      raise Exception("check source path and destination path")
```
Comment (mupputur) [inline Chaitanya_1/adb_trandfer_file.py:9]: Why you are raising an exception here ?  This code all the time throws an exception

File: Chaitanya_1/adb_trandfer_file.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#copy a file device to computer by using adb commands
+import os
+import subprocess
+def copy_a_file_device_to_computer(cmd_list):
```
Comment (mupputur) [inline Chaitanya_1/adb_trandfer_file.py:4]: Function design is not correct please use this way copy_a_file_device_to_computer(src_path, dest_path)

File: Chaitanya_1/adb_trandfer_file.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#copy a file device to computer by using adb commands
+import os
```
Comment (mupputur) [inline Chaitanya_1/adb_trandfer_file.py:2]: Please execute before you check-in with all possible scenarios

File: Chaitanya_1/adb_trandfer_file.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import subprocess
+def copy_a_file_device_to_computer(cmd_list):
+      try:
+            proc = subprocess.check_output(cmd).decode('utf-8')
+      except Exception as e:
+            print("Error :{}".format(str(e)))
```
Comment (mupputur) [inline Chaitanya_1/adb_trandfer_file.py:8]: Additional validation is required , really the file is copied in respective path or not

**Review by mupputur — CHANGES_REQUESTED** [review]
Please fix the comments

**Pull request merged**